content/scout/release-notes.md

```diff
@@ -14,7 +14,7 @@ for what's coming next.
 
 ## 2024 Q1
```

Comment:
Wondering if it makes more sense to call out as Q1 and then year?

```suggestion
## Q1 2024
```

Reply:
sure